packages/rich-text/src/register-format-type.js

```diff
@@ -14,6 +14,8 @@ import { store as richTextStore } from './store';
  * @property {string}        tagName     The HTML tag this format will wrap the
  *                                       selection with.
  * @property {boolean}       interactive Whether format makes content interactive or not.
+ * @property {boolean}       object      Whether the format represents an object (e.g., `img`, `br`),
```

Comment:
does object in this case here stand for "self closing tag"?

Reply:
As far as I understand, it's not a requirement, just good examples.

Let's say you've a custom format with `span` tagname that displays special emojis, it can/should be an object type.

cc @ellatrix

Reply:
Okay, so it means that the format does not allow any other custom text within where as non objects wrap around other text 🤔

Reply:
An object cannot wrap around other text/formats, but itself can be wrapped in other formats. Text and an inline image can be technically linked together.